src/DynamoCore/Library/FunctionDescriptor.cs

```diff
@@ -23,6 +23,11 @@ public interface IFunctionDescriptor
         string DisplayName { get; }
 
         /// <summary>
+        /// Name to create node
+        /// </summary>
```

Comment:
This needs a more descriptive summary and please indent to be consistent.  I'm still not quite sure why this is necessary.